src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/Tags/GaugeCondition.hpp

```diff
@@ -37,5 +49,53 @@ struct GaugeCondition : db::SimpleTag {
     return gauge_condition->get_clone();
   }
 };
+
+/// @{
+//// \brief Gauge condition \f$H_a\f$ and its spacetime derivative
+/// \f$\partial_b H_a\f$
+template <size_t Dim>
+struct GaugeAndDerivative
+    : ::Tags::Variables<
+          tmpl::list<::GeneralizedHarmonic::Tags::GaugeH<Dim, Frame::Inertial>,
+                     ::GeneralizedHarmonic::Tags::SpacetimeDerivGaugeH<
+                         Dim, Frame::Inertial>>> {
```

Comment:
I think it's mostly convention to have a simple tag with the same name as the compute tag... Honestly, it didn't cross my mind while writing this. I'll just remove because I agree it's sort of redundant...